build/release.mk

```diff
@@ -0,0 +1,6 @@
+RELEASE_SCRIPT_PATH ?= ./tools/release.sh
+
+# Example usage: make release version=0.11.0
+release:
+	@echo "=== $(PROJECT_NAME) === [ release          ]: Generating release."
+	./tools/release.sh $(version)
```

Comment:
Maybe use `RELEASE_SCRIPT_PATH` from above in place of the direct path.  What do you think?

Reply:
Oh yeah, I meant to do that (jet lag fever and such 😉). Thanks.

Reply:
Updated